el/templates/test/_tdd.html

```diff
@@ -0,0 +1,53 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<!doctype html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title><%= elementName %></title>
+
+  <script src="../../bower_components/webcomponentsjs/webcomponents.min.js"></script>
+  <link rel="import" href="../../bower_components/polymer/polymer.html">
+  <script src="../../bower_components/web-component-tester/browser.js"></script>
+  <script src="../../bower_components/test-fixture/test-fixture-mocha.js"></script>
+  <link rel="import" href="../../bower_components/test-fixture/test-fixture.html">
+
+  <!-- Step 1: import the element to test -->
+  <link rel="import" href="../elements/<%= elementName %>/<%= elementName %>.html">
+
+</head>
+<body>
+
+  <test-fixture id="basic">
+    <template>
+      <<%= elementName %>></<%= elementName %>>
+    </template>
+  </test-fixture>
+
+  <script>
+
+    suite('<%= elementName %> tests', function() {
+      var element;
+
+      setup(function() {
+        element = fixture('basic');
+      });
+
+      test('', function() {
+
+      });
+
+    });
+  </script>
+
+</body>
+</html>
+
```

Comment:
It's going to be annoying that we'll have to maintain this file here while all other files are in other repos.